modelforge/tests/test_remote.py

```diff
@@ -5,8 +5,6 @@
 
 from modelforge.utils.remote import *
 
-IN_GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") == "true"
-
 
 @pytest.fixture(scope="session")
 def prep_temp_dir(tmp_path_factory):
```

Comment:
might make sense to put this in a `conftest.py` file

Reply:
Ah, so this just got left in accidentally, as I was testing some different ways to mark tests for local vs. CI.